guide/additional-info/rest-api.md

```diff
@@ -4,128 +4,155 @@ REST APIs are extremely popular on the web and allow you to freely grab a site's
 
 If you've ever seen a music bot that accepts a YouTube query instead of just a video's URL, then you've seen a REST API in action. As a matter of fact, discord.js is made to use Discord's API. So, you've probably used an API yourself.
 
-## Using a REST API with node
+## Using a REST API with Node
 
-In these examples we are going to be using [Snekfetch](https://www.npmjs.com/package/snekfetch) which is a great library for making HTTP requests by one of the developers of discord.js. Discord.js actually uses Snekfetch which is part of the reason why it's ideal; it's already installed if you're using discord.js!
+In these examples we are going to be using [snekfetch](https://www.npmjs.com/package/snekfetch) which is a great library for making HTTP requests by one of the developers of discord.js. Discord.js actually uses snekfetch which is part of the reason why it's ideal; it's already installed if you're using discord.js!
 
-If you're not using discord.js you'll simply have to do the following to install Snekfetch.
+If you're not using discord.js you'll simply have to do the following to install snekfetch.
 
 ```bash
 npm install --save snekfetch
 ```
 
 ## Using Snekfetch
 
-Snekfetch is a promise-based request library with beautiful syntax. If you don't already know about promises, you should read up on them [here](/additional-info/async-await).
+Snekfetch is a promise-based request library with beautiful syntax. If you aren't already familiar with promises, you should read up on them [here](/additional-info/async-await).
 
 In this tutorial we'll be making a bot with 2 API based commands. The first will be using [random.cat](https://random.cat) and the other will use [Urban Dictionary](https://www.urbandictionary.com).
 
-<p class="tip">We're going to take advantage of [destructuring](/additional-info/es6-syntax?id=destructuring) in this tutorial because it just looks great.</tip>
+<p class="tip">We're going to take advantage of [destructuring](/additional-info/es6-syntax?id=destructuring) in this tutorial to maintain readability.</p>
```

Comment:
This could go inside the skeleton code section, between the paragraph and the codeblock, since that's where it'd make sense the most now.